### ISSUE TITLE
Mac: Shortcuts to groups/folders in a campaign point to main campaign root instead of the subfolder/group.

### ISSUE THREAD

**Issue by Duseylicious** [encounterplus/encounterplus#122]
In the Mac version, when I have a campaign (Tyranny of Dragons). Inside the campaign I have folders/groups for each chapter (Ch 11 The Tomb). If I press the shortcut button at the top of the screen, then the plus + button to add a shortcut, and then choose a Reference that is a sub folder (Ch 11 The Tomb), and save it, when I tap the newly created shortcut, it opens up the campaign root (Tyranny of Dragons) instead of the chapter subfolder (Ch 11 The Tomb).

Works as expected on iOS. (I noticed just now that there is no "shortcut" button in the context menu of folders, like there is for maps, encounters, etc.)

**Comment by jurex** [comment]
Hmm, cannot replicate this @Duseylicious, could you please check if you have correct slug defined in your group? Without the unique slug the group referencing won't work properly.

**Comment by Duseylicious** [comment]
Ok, the slug is definitely the problem - the interesting thing how it got here.

I went to check the slugs, and all the slugs that came from my iCloud backup from my iPad that have spaces in the names instead of being replaced with hyphens (whereas when I create them from scratch on the iPad or the Mac the spaces are converted to hyphens.)  The slug names also had capitals.  ![Screen Shot 2020-04-15 at 8 57 35 AM](https://user-images.githubusercontent.com/63576494/79359509-916bbb80-7ef7-11ea-8fab-ba323d9c3011.png)

I made a new campaign with groups that all had unique names.  Everything worked.  I fixed the slug on one of the problem groups and got rid of the spaces and caps, and then the shortcutting worked.  

So I'll go through and start fixing the slugs, but I just verified that all the slugs on the iPad are properly formatted and working, so maybe there is something going on that caused the change during the backup/restore process.
